pkg/master/master.go

```diff
@@ -72,14 +72,15 @@ import (
 
 // Config is a structure used to configure a Master.
 type Config struct {
-	Cloud             cloudprovider.Interface
-	EtcdHelper        tools.EtcdHelper
-	EventTTL          time.Duration
-	MinionRegexp      string
-	KubeletClient     client.KubeletClient
-	PortalNet         *net.IPNet
-	EnableLogsSupport bool
-	EnableUISupport   bool
+	Cloud              cloudprovider.Interface
+	EtcdHelper         tools.EtcdHelper
+	EventTTL           time.Duration
+	MinionRegexp       string
+	KubeletClient      client.KubeletClient
+	PortalNet          *net.IPNet
+	PortalServicePorts util.PortRange
```

Comment:
Needs a better name - this isn't really about portals it's about proxies.  Suggest ProxyPorts or NodeProxyPorts or ProxyPortRange or something.